BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -96,15 +96,48 @@ public struct AccessibilityElement: Element {
 
 extension Element {
```

Comment:
The file this is in has the `AccessibilityElement` bAse type; we should rename that too and add a typealias that's deprecated.

If we want to, I move deprecated types in Listable into their own file w/ date markers; so they're later easier to know about and remove: https://github.com/kyleve/Listable/blob/main/ListableUI/Sources/Deprecations.swift

Reply:
Since we'll be going with `accessibilityElement(...)` (at least for now) I'm going to stick with the existing structure.

Since I believe it's unlikely that consumers are currently instantiating `AccessibilityElement` directly I'll go ahead and just update those parameters to be required (to match `accessibilityElement(...)`).

- [ ] make parameters required on `AccessibilityElement`

Reply:
Actually I can deprecate the existing initializer and create a new one with parameters that match the order defined in `accessibilityElement`. I'll do that.